content/advanced/330_servicemesh_using_appmesh/add_nodegroup_fargate/create_fargate.md

```diff
@@ -0,0 +1,186 @@
+
+#### Enable Logging for Fargate
+
+Amazon EKS with Fargate supports a built-in log router, which means there are no sidecar containers to install or maintain. Apply a ConfigMap to your Amazon EKS cluster with a Fluent Conf data value that defines where container logs are shipped to.
+This logging ConfigMap has to be used in a fixed namespace called aws-observability has a cluster-wide effect, meaning that you can send application-level logs from any application in any namespace.
+
+In this workshop, we will show you how to use cloudwatch_logs to send logs from a workload running in an EKS on Fargate cluster to CloudWatch.
+
+First, create the dedicated aws-observability namespace and the ConfigMap for Fluent Bit
+```bash
+envsubst < ./deployment/fluentbit-config.yaml | kubectl apply -f -
+```
+{{< output >}}
+namespace/aws-observability created
+configmap/aws-logging created
+{{< /output >}}
+
+Next, verify if the Fluent Bit ConfigMap is set up correctly
+```bash
+kubectl -n aws-observability get cm
+```
+{{< output >}}
+NAME          DATA   AGE
+aws-logging   1      18s
+{{< /output >}}
+
+With Fluent Bit set up we next need to give it the permission to write to CloudWatch. We do that by first downloading the policy locally:
+```bash
+curl -o permissions.json \
+     https://raw.githubusercontent.com/aws-samples/amazon-eks-fluent-logging-examples/mainline/examples/fargate/cloudwatchlogs/permissions.json
+```
+{{< output >}}
+% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                Dload  Upload   Total   Spent    Left  Speed
+100   215  100   215    0     0   1023      0 --:--:-- --:--:-- --:--:--  1023
+{{< /output >}}
+And next we create the policy and attach it to the pod execution role of your EKS on Fargate cluster:
+```bash
+aws iam create-policy \
+        --policy-name FluentBitEKSFargate \
+        --policy-document file://permissions.json 
+```
+{{< output >}}
+{
+    "Policy": {
+        "PolicyName": "FluentBitEKSFargate", 
+        "PermissionsBoundaryUsageCount": 0, 
+        "CreateDate": "2021-02-04T07:11:07Z", 
+        "AttachmentCount": 0, 
+        "IsAttachable": true, 
+        "PolicyId": "ANPAV45SCB72QX3SZN2RS", 
+        "DefaultVersionId": "v1", 
+        "Path": "/", 
+        "Arn": "arn:aws:iam::405710966773:policy/FluentBitEKSFargate", 
+        "UpdateDate": "2021-02-04T07:11:07Z"
+    }
+}
+{{< /output >}}
+
+Attach the Policy to the Pod Execution Role of your EKS on Fargate cluster                                            
+```bash
+export PodRole=$(aws eks describe-fargate-profile --cluster-name eksworkshop-eksctl --fargate-profile-name fargate-productcatalog | sed -n 's/^.*role\/\(.*\)".*$/\1/ p')
```

Comment:
I need to as otherwise, it gives this "arn:aws:iam::$ACCOUNT_ID:role/eksctl-eksworkshop-eksctl-FargatePodExecutionRole-B57GSVCY9Y6O" and I need this eksctl-eksworkshop-eksctl-FargatePodExecutionRole-B57GSVCY9Y6O